maquetado/scss/detail.scss

```diff
@@ -0,0 +1,35 @@
+.card {
+  display: flex;
+  flex-flow: column wrap;
+}
+
+.card-description {
+  margin: 18px 6px;
+}
+
+.default-text {
+  color: #828282;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.information {
+  color: #000;
+  font-size: 20px;
+  font-weight: bold;
```

Comment:
For example look that for `.information .title and .small-title` classes you are define font-weight bold, what do you think if try to use only one class?